src/frequenz/sdk/actor/power_distributing.py

```diff
@@ -91,6 +91,34 @@ def update_retry(self, timeout_sec: float) -> None:
         """
         self._timeout_sec = timeout_sec
 
+    def get_working_subset(self, components_ids: Set[int]) -> Set[int]:
+        """Get subset of batteries that are not marked as broken.
+
+        If all given batteries are broken, then mark them as working and return them.
+        This is temporary workaround to not block user command.
+
+        Args:
+            components_ids (Set[int]): set of component ids
+
+        Returns:
+            Set[int]: subset of given components_ids with working components.
```

Comment:
we are not specifying types for args or returns anymore.

Reply:
Done

Reply:
Done